doc/Makefile

```diff
@@ -29,5 +29,13 @@ html: source Makefile
 %.rst: %.rst.py %.rst.jinja
 	PYTHONPATH="./:$$PYTHONPATH" python3 $< > $@
 
+%.rst: %.rst.in
+	MIN_AUTOCONF_VERSION=$$(sed -n -e 's/\s*AC_PREREQ(\[\(.*\)\])/\1/p' ../configure.ac) ; \
+	MIN_PYTHON_VERSION=$$(sed -n -e 's/\s*AM_PATH_PYTHON(\[\(.*\)\])/\1/p' ../configure.ac) ; \
```

Comment:
Your regexp should be
`AC_PREREQ(\(\[.*\]\))` (i.e. square and round brackets swapped)
and equivalent for Python, as in the `configure.ac` the versions are 
`AC_PREREQ([2.69])` and `AM_PATH_PYTHON([3.9])`

Reply:
@MJKirk's comment is correct for ``sed -E``. I am using ``sed -e``, in which the roles of ``\(\)`` and ``()`` are interchanged. :upside_down_face:

Reply:
You win this round